openprescribing/frontend/management/commands/send_monthly_alerts.py

```diff
@@ -165,6 +166,27 @@ def send_search_bookmark_email(self, search_bookmark, now_month):
         except bookmark_utils.BadAlertImageError as e:
             logger.exception(e)
 
+    def send_all_england_alerts(self, options):
+        # The `send_all_england_alerts` command doesn't respect the same set of
+        # options that this command does, so we only invoke it for certain
+        # limited sets of options
+        set_options = {k: v for (k, v) in options.items() if v is not None}
```

Comment:
Sometimes, I miss Ruby.

Reply:
Yes, I think it was the ergonomics of operations like this which originally drove Gary Bernhardt from Python to Ruby.